src/main/java/org/apache/commons/lang3/time/StopWatch.java

```diff
@@ -671,4 +686,622 @@ public void unsplit() {
         splitState = SplitState.UNSPLIT;
     }
 
+    /**
+     * Take the time of the exectution of a given {@link FailableSupplier}.
+     *
+     * <p>
+     * <em>Take the time of the execution of given {@link FailableSupplier}</em>
+     * </p>
+     * <pre>{@code
+     * final StopWatch watch = StopWatch.create();
+     * String result = watch.get(() -> "A");
+     * }</pre>
+     *
+     * <p>
+     *  <em>Note:</em>
+     * </p>
+     * <ul>
+     *     <li>Be carefull if you pass a {@link FailableSupplier} to other methods, as the wrapped instance depends on
```

Comment:
Not all typos were fixed, there are still a couple of JavaDoc with "carefull" instead of "careful".

Reply:
Yep, good catch.

Reply:
Sorry, I thought that I have done this. Sorry.